content/guides/astro.md

```diff
@@ -39,4 +200,13 @@ clever env set CC_PRE_BUILD_HOOK "npm install && npm run astro telemetry disable
 clever env set CC_POST_BUILD_HOOK "npm run build"
 ```
 
-{{% content/git-push %}}
\ No newline at end of file
+{{% content/git-push %}}
+
+## Learn more
+
+{{< cards >}}
+  {{< card link="../doc/applications/javascript/nodejs" title="Deploy a Node.js application" subtitle="Learn more on deploying a Node.js application on Clever Cloud" icon="node" >}}
+  {{< card link="../doc/applications/static" title="Deploy a Static application" subtitle="Learn more on deploying a Static application on Clever Cloud" icon="feather" >}}
+  {{< card link="https://docs.astro.build/en/getting-started/" title="Learn Astro" subtitle="Astro full documentation" icon="rocket-launch" >}}
```

Comment:
```suggestion
  {{< card link="https://docs.astro.build/" title="Learn Astro" subtitle="Astro full documentation" icon="rocket-launch" >}}
```

The more generic an external link is, the less we'll need to update it ^^

Reply:
true!